packages/grid/_modules_/grid/components/toolbar/DefaultToolbar.tsx

```diff
@@ -0,0 +1,29 @@
+import { useContext } from 'react';
+import * as React from 'react';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { ApiContext } from '../api-context';
+import { GridToolbar } from '../containers/GridToolbar';
+import { ColumnsToolbarButton } from './ColumnsToolbarButton';
+import { DensityMenuSelector } from './DensityMenuSelector';
+import { FilterToolbarButton } from './FilterToolbarButton';
+
+export function DefaultToolbar() {
```

Comment:
The usage of *"Default"* is weird (compared to the rest of the codebase, Autocomplete, DatePicker, etc.). It's already implicit, all the components are default as they can be customized. 

How about?
```suggestion
export function GridHeaderToolbar() {
```

x all the other cases where we remove "Default"